### DISCUSSION TITLE
Discussion about more test runnables

### DISCUSSION THREAD

**Discussion by CsullogBeni** [CsullogBeni/szofttech#28] [Ideas]
## 👋 Hello!
We should start come up with more ideas 💡 about executables.
  ➡️ Come up with scripts that easy to implement.
  ➡️ These executables must have various arguments.
  ➡️ These executables should log the outputs.

Build together 💪.

  To get started, comment below if you have any idea.

**Comment by laczkozsofi** [comment]
An idea:
Min/Max selection script
Arguments:
- some numbers (user decide how many)
- optional flag if we want min, default would be max

**Reply by CsullogBeni** [reply]
Sounds good. If you want to implement it, please make an issue, and you can start it.

**Comment by laczkozsofi** [comment]
An other idea, just to something with strings:
Alphabetical order script
Arguments:

- optional flag: asc or desc (default asc)
- obligatory: list of string (for example seperated by comma)

Result: print the alphabetical ordered list of the strings
